### PR TITLE
profile page and code clean up

### PR DESCRIPTION
**Pull request by Abhijay007**
**Describe the changes you've made**

- Removed unwanted footer section from the profile page 
- Added pages links to the main footer section 
- Removed extra dark mode toggle from the profile page 
- Added/modified header component of the profile page according to the Figma design.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`NA`

**Additional context (OPTIONAL)**

screencapture-localhost-3000-profile-2021-06-15-21_23_45

**Checklist**

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**
Preview: Deploy preview link here with the appropriate route

### CODE REVIEW

File: src/components/ProfilePageComponents/HeaderComponent.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import { useHistory } from "react-router-dom";
+const useStyles = makeStyles((theme) => ({
+  editbutton: {
+    marginTop: "5px",
+    background: "tansparent",
+    textTransform: "none",
+    padding: "0 16px 0 16px",
+    border: "2px solid #FFFFFF",
+    boxSizing: "border-box",
+    borderRadius: "10px",
```
Comment (Abhishek-kumar09) [inline src/components/ProfilePageComponents/HeaderComponent.js:12]: We are now placing useStyles below the functional component to help digging through code better.

**Comment by Abhishek-kumar09** [comment]
Please review @adarsh-technocrat

**Comment by adarsh-technocrat** [comment]
Sure @Abhishek-kumar09   Actually this PR requires changes  I'll add review comments for that!

**Comment by adarsh-technocrat** [comment]
@Abhijay007  please make this PR draft!

**Comment by Abhijay007** [comment]
Resolved in #368

**Pull request closed without merging**